### PR TITLE
trivial: random fixes

### PR DESCRIPTION
**Pull request by oxij**
See commit messages.

### CODE REVIEW

**Comment by oxij** [comment]
@GrahamcOfBorg build cmake_2_8 datefudge mdf2iso mr

**Comment by oxij** [comment]
vlc is fat on deps, but I built both versions (with and without qt) locally, so this should be fine. The change is a noop for the default vlc config anyway.

File: pkgs/applications/version-management/mr/default.nix [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 stdenv.mkDerivation rec {
-  version = "1.20180726";
   name = "mr-${version}";
+  version = "1.20180726";
 
   src = fetchgit {
     url = "git://myrepos.branchable.com/";
-    rev = "refs/tags/" + version;
-    sha256 = "15i9bs2i25l7ibv530ghy8280kklcgm5kr6j86s7iwcqqckd0czp";
+    rev = "0ad7a17bb455de1fec3b2375c7aac72ab2a22ac4";
```
Comment (timokau) [inline pkgs/applications/version-management/mr/default.nix]: Why? The version is tagged.

**Comment by oxij** [comment]
> Why? The version is tagged.

So that the hash won't change if the tag moves again.

**Comment by timokau** [comment]
If they move their tags you should probably add that as a comment. Pretty ironic that some project that has to do with version control moves the git tags.

**Review by 7c6f434c — APPROVED** [review]

**Comment by oxij** [comment]
Added a comment.

**Comment by timokau** [comment]
Looks like vlc [never worked](https://hydra.nixos.org/job/nixpkgs/trunk/vlc.aarch64-linux/all?page=18) on aarch64. We should always fix those cases when we see them. Can you please add a commit disabling the aarch64 platform for the relevant dependency?

**Comment by c0bw3b** [comment]
No it has worked [in the past](https://hydra.nixos.org/job/nixos/release-18.03-aarch64/nixpkgs.vlc.aarch64-linux).
Current failure is only because of `live555` and #48825 should fix it.

**Comment by timokau** [comment]
Oh thats good. I assumed the trunk job would show all past builds.

**Review by timokau — APPROVED** [review]

**Pull request merged**